arch/power/clang_workaround.h

```diff
@@ -0,0 +1,98 @@
+/* Helper functions to work around issues with clang builtins
+ * Copyright (C) 2021 IBM Corporation
+ *
+ * Authors:
+ *   Daniel Black <daniel@linux.vnet.ibm.com>
+ *   Rogerio Alves <rogealve@br.ibm.com>
+ *   Tulio Magno Quites Machado Filho <tuliom@linux.ibm.com>
+ *
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#ifndef CLANG_WORKAROUNDS_H
```

Comment:
Use similar to fallback_builtins.h `POWER_BUILTINS_H`?

Reply:
Done.